app/models/site_banner.rb

```diff
@@ -7,6 +7,14 @@ class SiteBanner < ApplicationRecord
   has_one_attached :background_image
   belongs_to :publisher
 
+  LOGO = "logo".freeze
+  LOGO_DIMENSIONS = [148,148]
+  LOGO_UNIVERSAL_FILE_SIZE = 20_000
+
+  BACKGROUND = "background".freeze
+  BACKGROUND_DIMENSIONS = [900,176]
+  BACKGROUND_UNIVERSAL_FILE_SIZE = 60_000
```

Comment:
can you add some comments here, above, and for `MAX_IMAGE_SIZE` that specify the unit